pkg/scheduler/plugins/nodeorder/nodeorder.go

```diff
@@ -335,13 +323,8 @@ func (pp *nodeOrderPlugin) OnSessionOpen(ssn *framework.Session) {
 			return nil, err
 		}
 
-		selectorSpreadScores, err := selectorSpreadScore(selectorSpread, state, task.Pod, nodes, weight.selectorSpreadWeight)
-		if err != nil {
-			return nil, err
-		}
-
 		for _, node := range nodes {
-			nodeScores[node.Name] = podAffinityScores[node.Name] + nodeTolerationScores[node.Name] + podTopologySpreadScores[node.Name] + selectorSpreadScores[node.Name]
```

Comment:
What's the compatibility policy of volcano with kubernets version support? User with lower kubernetes version than 1.29 will miss the plugin.

Reply:
After review, the plugin of  SelectorSpread can be deleted for the following reasons: First, k8s provides the PodTopologySpread plugin to replace the SelectorSpread plugin in version 1.19; secondly, volcano introduced both plugins at the same time, and the selector plugin is not used by default, but the podtopology plugin is used.